app/view_models/prior_authority/v1/quote.rb

```diff
@@ -9,6 +9,8 @@ class Quote < BaseWithAdjustments
       attribute :id, :string
       attribute :cost_type, :string
       attribute :item_type, :string, default: 'item'
+      attribute :cost_item_type, :string
+      attribute :cost_multiplier, :decimal, precision: 10, scale: 5, default: 1
```

Comment:
why does the scale need to be greater than for cost per hour/item (scale: 2)?

Reply:
scratch that i assume that is because 0.001, although is that  not `scale: 3`. but sounds ok anyway